### ISSUE TITLE
After rename of openerp directory to odoo, setup.py no longer functions

### ISSUE THREAD

**Issue by NL66278** [odoo/odoo#13582]
Impacted versions: master (v10)

Steps to reproduce: Create an anybox buildout recipe for installation of master and do a buildout

Current behavior: Crash occurs with following message:
While:
  Installing odoo.

An internal error occurred due to a bug in either zc.buildout or in a
recipe being used:
```
Traceback (most recent call last):
  File "/home/openeyedev/var/projects/oe_political_party_10/buildout_acp/eggs/zc.buildout-2.5.3-py2.7.egg/zc/buildout/buildout.py", line 1995, in main
    getattr(buildout, command)(args)
  File "/home/openeyedev/var/projects/oe_political_party_10/buildout_acp/eggs/zc.buildout-2.5.3-py2.7.egg/zc/buildout/buildout.py", line 666, in install
    installed_files = self[part]._call(recipe.install)
  File "/home/openeyedev/var/projects/oe_political_party_10/buildout_acp/eggs/zc.buildout-2.5.3-py2.7.egg/zc/buildout/buildout.py", line 1410, in _call
    return f()
  File "/home/openeyedev/var/projects/oe_political_party_10/buildout_acp/anybox.recipe.odoo/anybox/recipe/odoo/base.py", line 1098, in install
    self.read_openerp_setup()
  File "/home/openeyedev/var/projects/oe_political_party_10/buildout_acp/anybox.recipe.odoo/anybox/recipe/odoo/base.py", line 642, in read_openerp_setup
    ('.py', 'r', imp.PY_SOURCE))
EnvironmentError: Problem while reading Odoo setup.py:
```
Expected behavior: Odoo can be installed with buildout

Video/Screenshot link (optional):

**Comment by NL66278** [comment]
Diagnose:
line 11 of setup.py refers to the openerp directory, that is now the odoo directory:
 11 execfile(join(dirname(__file__), 'openerp', 'release.py'))  # Load release variables

**Comment by lmignon** [comment]
See https://github.com/odoo/odoo/pull/13413

**Comment by Yenthe666** [comment]
@mart-e can you have a look at this one? @lmignon already provided a fix at https://github.com/odoo/odoo/pull/13413

**Comment by lmignon** [comment]
@sbidoul  already provided a fix at #13413

**Comment by mart-e** [comment]
I believe fixed at #13609
